store/tikv/kv_test.go

```diff
@@ -0,0 +1,25 @@
+package tikv
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/config"
+)
+
+type testTiKVDriverSuite struct {
+	OneByOneSuite
+	store *tikvStore
+}
+
+var _ = Suite(&testTiKVDriverSuite{})
+
+func (s *testTiKVDriverSuite) TestSetDefaultAndOptions(c *C) {
+	d := Driver{}
+	security := config.Security{ClusterSSLCA: "test"}
+	d.setDefaultAndOptions(WithSecurity(security))
+
+	defaultCfg := config.GetGlobalConfig()
+	c.Assert(d.security, DeepEquals, security)
+	c.Assert(d.tikvConfig, DeepEquals, defaultCfg.TiKVClient)
+	c.Assert(d.txnLocalLatches, DeepEquals, defaultCfg.TxnLocalLatches)
+	c.Assert(d.pdConfig, DeepEquals, defaultCfg.PDClient)
+}
```

Comment:
Could you also add an assertion to assert that the `security` in global config is not modified?

Reply:
added in https://github.com/pingcap/tidb/pull/22170/commits/84e21944c7f645d53664b1aec7b397cb5e0baa9c